autorelease/tag.py

```diff
@@ -48,16 +52,24 @@ def process_issue(
     # is necessary because github gives us back an issue object, but it
     # doesn't contain all of the PR info.
     pull = gh.get_url(issue["pull_request"]["url"])
+    repo_full_name = pull["base"]["repo"]["full_name"]
 
     # Determine language.
-    lang = common.guess_language(gh, pull["base"]["repo"]["full_name"])
+    lang = common.guess_language(gh, repo_full_name)
 
     # As part of the migration to release-please tagging, cross-reference the
     # language against an allowlist to allow migrating language-by-language.
     if lang not in LANGUAGE_ALLOWLIST:
         result.skipped = True
         result.print(f"Language {lang} not in allowlist, skipping.")
         return
+    # As part of the migration to release-please tagging, cross-reference the
+    # repository against a denylist to allow migrating repo-by-repo.
+    for denied_repo in REPO_DENYLIST:
+        if denied_repo in repo_full_name:
```

Comment:
Unlikely for this single exception, but this could match more than desired. Consider an exact match for the full `<org>/<repo>`.

Reply:
Yes, also better, done.